<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
    <widget class="QLabel" name="label_maxe1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max( e1 ) = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mine1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min( e1 ) =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxe2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max( e2 ) = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mine2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min( e2 ) =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max(S) =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_minS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min(S) =</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="2">
      <widget class="QLineEdit" name="lineEdit_f">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEdit_k">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>m</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>k</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEdit_m">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string> U'(0) </string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_V1">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string> U(0)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_V2">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>f</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_h">
       <property name="text">
        <string>0.01</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_N">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>N</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Таблица</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>451</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>x</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>h</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>V1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>V2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>^V1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>^V2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>e1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>e2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>S</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Enter">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graph" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graphPortret" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clear">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Portret">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>470</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Фаз портрет</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Control">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>  Контроль</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_End">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> Граница</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_X">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Eb">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>45</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1e-7</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
